src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.awt.datatransfer.StringSelection;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Duke {
+    static void printHello() {
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
+    }
+    static void printTaskList(int index, Task[] Tasklist) {
```

Comment:
I would suggest changing the Tasklist array name to follow the Java coding conventions of naming, - camel case and arrays should show that they contain multiple elements, such as tasks